lib/ts/recipe/utils.ts

```diff
@@ -0,0 +1,122 @@
+import * as jose from "jose";
```

Comment:
Please move this file one level up and rename it to something like `thirdpartyutils`

Reply:
done